Dockerfile.repo

```diff
@@ -90,14 +89,13 @@ WORKDIR $ACAS_HOME
 
 ENV R_LIBS $ACAS_HOME/r_libs
 RUN mkdir $ACAS_HOME/r_libs
-ENV PATH /usr/pgsql-13/bin/:$PATH
+# ENV PATH /usr/pgsql-13/bin/:$PATH
 
 COPY	DESCRIPTION $ACAS_HOME
 COPY	R/installation.R $ACAS_HOME
 RUN chown -R runner:runner $ACAS_HOME
-RUN yum install -y nlopt nlopt-devel sudo
+
```

Comment:
Now leaving Description and Install.R on the build because I needed it for debugging, I think its ok to leave this there as its minimal in size.